js/viz/docs/docSankey.js

```diff
@@ -315,6 +315,18 @@ var dxSankey = {
         * @type object
         */
         hoverStyle: {
+            /**
```

Comment:
While we're at it: quotes around the value are missing at lines [#175](https://github.com/DevExpress/DevExtreme/pull/5084/files#diff-fbfbbcecd00762282855c080e462d19eR175) and [#309](https://github.com/DevExpress/DevExtreme/pull/5084/files#diff-fbfbbcecd00762282855c080e462d19eR309)